pages/settings/devices.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <settings-page page-title="settings.devices.devices">
+    <template #actions>
+      <v-btn
+        v-if="devices"
+        color="error"
+        class="ml-a"
+        @click="deleteAllDevices"
+      >
+        {{ $t('settings.devices.deleteAll') }}
+      </v-btn>
+      <v-btn
+        ref="noreferrer noopener"
+        href="https://jellyfin.org/docs/general/server/devices.html"
+        target="_blank"
+      >
+        {{ $t('settings.help') }}
+      </v-btn>
+    </template>
+    <template #content>
+      <v-col>
+        <v-data-table
+          :headers="headers"
+          :items="devices"
+          @click:row="setSelectedDevice"
+        >
+          <!-- eslint-disable-next-line vue/valid-v-slot -->
+          <template #item.DateLastActivity="{ item }">
+            <p class="text-capitalize-first-letter mb-0">
+              {{
+                $dateFns.formatRelative(
+                  $dateFns.parseJSON(item.DateLastActivity),
+                  new Date()
+                )
+              }}
+            </p>
+          </template>
+        </v-data-table>
+      </v-col>
+      <v-dialog v-model="deviceInfoDialog" width="fit-content">
+        <selected-device-info
+          v-if="selectedDevice.Name"
+          :selected-device="selectedDevice"
+          :is-dialog="true"
+          @close-dialog="closeDialog"
+          @delete-selected="deleteSelectedDevice"
+        />
```

Comment:
We can also wrap all this component in a transition wrapper so it gets an animation: https://vuetifyjs.com/en/api/v-fade-transition/

Reply:
It already has a transition since it is inside a v-dialog